src/WebpackAssetsManifest.js

```diff
@@ -835,7 +835,10 @@ class WebpackAssetsManifest
       return true;
     }
 
-    return has(this, 'compiler.outputFileSystem') && this.compiler.outputFileSystem.constructor.name === 'MemoryFileSystem';
+    return has(this, 'compiler.outputFileSystem')
+      && this.compiler.outputFileSystem !== null
+      && this.compiler.outputFileSystem.constructor
+      && this.compiler.outputFileSystem.constructor.name === 'MemoryFileSystem';
```

Comment:
I ran `npm test` on your branch and the tests failed due to this function returning `undefined` when `constructor` was not defined.

This is a simpler replacement that always returns a `boolean` and makes the tests pass.

```suggestion
    return get(this, 'compiler.outputFileSystem.constructor.name') === 'MemoryFileSystem';
```

Reply:
test is fixed. sorry about inattentiveness i was in airport making previous commit with absolutely terrible Internet connection